avocado/core/nrunner.py

```diff
@@ -542,11 +542,10 @@ class PythonUnittestRunner(BaseRunner):
 
     Runnable attributes usage:
 
-     * uri: a "dotted name" that can be given to Python standard
-            library's :meth:`unittest.TestLoader.loadTestsFromName`
-            method. While it's not enforced, it's highly recommended
-            that this is "a test method within a test case class" within
-            a test module.  Example is: "module.Class.test_method".
+     * uri: a test reference (string that can be resolved into one or more
```

Comment:
This change is correct in the sense that what must be given to the runner as an uri is indeed an Avocado-style test reference (`path/to/file.py:Class:test_method`) and not a Python unittest-like "dotted name. But, I would remove the "string that can be resolved into one or more tests", because in a natural usage, a resolver will return a name that is about a single test indeed, and mentioning multiple tests here just adds noise IMO.